codegen/can_messages.asciipb

```diff
@@ -104,6 +104,28 @@ msg {
 
 # IDs: 11-17 Reserved
 
+msg {
+  id: 14
+  source: SOLAR_5_MPPTS
+  target: "TELEMETRY"
+  msg_name: "relay current 5 mppts"
```

Comment:
A better name might be possible for these, since the message doesn't really have anything to do with the current - maybe "relay open ok 5 mppts" (and "6 mppts" for the other one)?

Reply:
done